### PR TITLE
[Substrait to Velox] Code style cleanup

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**

### CODE REVIEW

File: velox/substrait/SubstraitParser.h [hunk 1]
```diff
@@ -86,12 +86,12 @@ class SubstraitParser {
...
   /// Map the Substrait function key word into Velox function key word.
-  std::string mapToVeloxFunction(const std::string& subFunc) const;
+  std::string mapToVeloxFunction(const std::string& substraitFunction) const;
 
  private:
-  /// Used for mapping Substrait function key words into Velox functions' key
-  /// words. Key: the Substrait function key word, Value: the Velox function key
-  /// word. For those functions with different names in Substrait and Velox,
+  /// A map used for mapping Substrait function key words into Velox functions'
+  /// key words. Key: the Substrait function key word, Value: the Velox function
```
Comment (mbasmanova) [inline velox/substrait/SubstraitParser.h:93]: typo: key words - keywords, key word -> keyword

What is Velox function keyword? It looks like this is just a mapping from Substrait function names to Velox function names, no?

File: velox/substrait/SubstraitToVeloxExpr.cpp [hunk 2]
```diff
@@ -52,39 +52,48 @@ SubstraitVeloxExprConverter::toVeloxExpr(
...
-      substraitParser_.findVeloxFunction(functionMap_, functionId);
-  auto substraitType = substraitParser_.parseType(sFunc.output_type());
-  auto veloxType = toVeloxType(substraitType->type);
+  const auto& veloxFunction = substraitParser_.findVeloxFunction(
+      functionMap_, substraitFunc.function_reference());
+  const auto& veloxType = toVeloxType(
+      substraitParser_.parseType(substraitFunc.output_type())->type);
+
+  // Omit alias because name change is not needed.
+  if (veloxFunction == "alias") {
```
Comment (mbasmanova) [inline velox/substrait/SubstraitToVeloxExpr.cpp:68]: I don't believe there is a function "alias" in Velox. What's the scenario where you hit this code path? Is there a test?

**Review by mbasmanova — COMMENTED** [review]
@zhejiangxiaomai The cleanup is nice. Thank you. Some questions below.

The PR title should be written as "[Substrait to Velox] Code style cleanup"

File: velox/substrait/SubstraitToVeloxExpr.cpp (hunk 2 above)
Comment (zhejiangxiaomai) [reply velox/substrait/SubstraitToVeloxExpr.cpp:68]: SparkSql has "alias" function，there is an example
`wordCounts = textFile.select(explode(split(textFile.value, "\s+")).alias("word")).groupBy("word").count()` . So we add this code.

File: velox/substrait/SubstraitParser.h (hunk 1 above)
Comment (zhejiangxiaomai) [reply velox/substrait/SubstraitParser.h]: yes, just a mapping. I have corrected the grammar errors

File: velox/substrait/SubstraitToVeloxExpr.cpp [hunk 3]
```diff
@@ -52,39 +52,48 @@ SubstraitVeloxExprConverter::toVeloxExpr(
...
+    VELOX_CHECK(params.size() == 1, "Alias expects one parameter.");
```
[the remark this answers is not in the document]
Comment (zhejiangxiaomai) [reply velox/substrait/SubstraitToVeloxExpr.cpp]: ok，I hava fixed it.

File: velox/substrait/SubstraitToVeloxExpr.cpp [hunk 4]
```diff
@@ -52,39 +52,48 @@ SubstraitVeloxExprConverter::toVeloxExpr(
...
-  auto veloxFunction =
-      substraitParser_.findVeloxFunction(functionMap_, functionId);
-  auto substraitType = substraitParser_.parseType(sFunc.output_type());
-  auto veloxType = toVeloxType(substraitType->type);
+  const auto& veloxFunction = substraitParser_.findVeloxFunction(
+      functionMap_, substraitFunc.function_reference());
+  const auto& veloxType = toVeloxType(
+      substraitParser_.parseType(substraitFunc.output_type())->type);
+
+  // Some engines like SparkSQL has alias function, we omit it because name change is not needed.
```
Comment (mbasmanova) [inline velox/substrait/SubstraitToVeloxExpr.cpp:67]: Actually, this should be done when generating the plan. Redundant transformation like "alias" should be removed.

Let's remove this change. It doesn't belong to "cleanup" PR anyway.

**Comment by mbasmanova** [comment]
@zhejiangxiaomai Would you sign the CLA? Without it I cannot merge this PR.

**Comment by zhejiangxiaomai** [comment]
Hi, @mbasmanova<https://github.com/mbasmanova>, my manager yang, Binwei has Helped me apply CLA this afternoon, Maybe I will get CLA  tomorrow.  I will tell you, when I get CLA.

**Comment by zhejiangxiaomai** [comment]
> 
Hi, [@mbasmanova](https://github.com/mbasmanova)<https://github.com/mbasmanova>, my manager yang, Binwei has Helped me apply CLA this afternoon, Maybe I will get CLA  tomorrow.  I will tell you, when I get CLA.

**Comment by mbasmanova** [comment]
@zhejiangxiaomai Thanks. Ping me when the CLA is signed and I'll merge the PR. Thank you for contributing.

**Comment by zhejiangxiaomai** [comment]
> @zhejiangxiaomai Thanks. Ping me when the CLA is signed and I'll merge the PR. Thank you for contributing.

Hi,@mbasmanova CLA is signed, please merge this code.

**Comment by mbasmanova** [comment]
Thank you, @zhejiangxiaomai . The PR should get merged soon.

**Pull request closed without merging**